Makefile

```diff
@@ -34,8 +34,6 @@ ifndef VERSION
 endif
 
 	perl -i -pe 's/OMERO_VERSION=(\S+)/OMERO_VERSION=$(VERSION)/' Dockerfile
```

Comment:
Since I cannot spot any `OMERO_VERSION` left in `Dockerfile`, these changes might make the `Makefile` irrelevant and it can probably cleaned up as a follow-up.

Reply:
I could not spot it but I kept it "just in case"
With the automation option proposed in another PR, the ``Makefile`` will become totally irrelevant